listenbrainz/webserver/static/js/src/user-feed/UserFeed.tsx

```diff
@@ -327,9 +329,94 @@ export default class UserFeedPage extends React.Component<
     this.setState({ recordingFeedbackMap: newFeedbackMap });
   };
 
+  deleteFeedEvent = async (event: TimelineEvent) => {
+    const { currentUser, APIService } = this.context;
+    const { newAlert } = this.props;
+    const { events } = this.state;
+    if (
+      event.event_type === EventType.RECORDING_RECOMMENDATION ||
+      event.event_type === EventType.NOTIFICATION
+    ) {
+      try {
+        const status = await APIService.deleteFeedEvent(
+          event.event_type,
+          currentUser.name,
+          currentUser.auth_token as string,
+          event.id
+        );
+        if (status === 200) {
+          newAlert(
+            "success",
+            "Successfully deleted!",
+            <>Successfully deleted!</>
+          );
+          const new_events = _remove(events, (element) => {
+            // Making sure the event that is getting deleted is either a recommendation or notification
+            // Since, recommendation and notification are in same db, and might have same id as a pin
+            // Similarly we later on filter by id and event_type for pin deletion
+            return !(
+              element?.id === event.id &&
+              (element.event_type === EventType.RECORDING_RECOMMENDATION ||
+                element.event_type === EventType.NOTIFICATION)
+            );
+          });
+          this.setState({ events: new_events });
+        }
+      } catch (error) {
+        newAlert(
+          "danger",
+          "Could not delete event",
+          <>
+            Something went wrong when we tried to delete your event, please try
+            again or contact us if the problem persists.
+            <br />
+            <strong>
+              {error.name}: {error.message}
+            </strong>
+          </>
+        );
+      }
+    } else if (event.event_type === EventType.RECORDING_PIN) {
+      try {
+        const status = await APIService.deletePin(
+          currentUser.auth_token as string,
+          event.id as number
+        );
+        if (status === 200) {
+          newAlert(
+            "success",
+            "Successfully deleted!",
+            <>Successfully deleted!</>
```

Comment:
I realize the message should be optional for cases like this where we don't have much else to say.
Let's try not to duplicate the title, and pass an empty string instead.

There's another instance of the same thing in this file.